Common subdirectories: terminus-font-4.49.1.orig/alt and terminus-font-4.49.1/alt
Common subdirectories: terminus-font-4.49.1.orig/bin and terminus-font-4.49.1/bin
Common subdirectories: terminus-font-4.49.1.orig/dup and terminus-font-4.49.1/dup
diff -u terminus-font-4.49.1.orig/ter-u14b.bdf terminus-font-4.49.1/ter-u14b.bdf
--- terminus-font-4.49.1.orig/ter-u14b.bdf	2023-09-11 22:10:29.524309844 +0300
+++ terminus-font-4.49.1/ter-u14b.bdf	2023-09-11 22:10:51.234346648 +0300
@@ -747,9 +747,9 @@
 BITMAP
 00
 00
-7C
-C6
-C6
+10
+38
+6C
 C6
 C6
 FE
@@ -2722,8 +2722,8 @@
 30
 18
 00
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -2743,8 +2743,8 @@
 18
 30
 00
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -2764,8 +2764,8 @@
 38
 6C
 00
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -2785,8 +2785,8 @@
 76
 DC
 00
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -2806,8 +2806,8 @@
 6C
 6C
 00
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -2827,8 +2827,8 @@
 38
 6C
 38
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -2847,8 +2847,8 @@
 BITMAP
 00
 00
-7E
-D8
+3E
+78
 D8
 D8
 FE
@@ -4065,8 +4065,8 @@
 BITMAP
 7C
 00
-7C
-C6
+38
+6C
 C6
 C6
 C6
@@ -4108,8 +4108,8 @@
 6C
 38
 00
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -4149,8 +4149,8 @@
 BITMAP
 00
 00
-7C
-C6
+38
+6C
 C6
 C6
 C6
@@ -6964,8 +6964,8 @@
 6C
 38
 00
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -7131,8 +7131,8 @@
 BITMAP
 7C
 00
-7E
-D8
+3E
+78
 D8
 D8
 FE
@@ -7489,8 +7489,8 @@
 0C
 18
 00
-7E
-D8
+3E
+78
 D8
 D8
 FE
@@ -8370,9 +8370,9 @@
 BITMAP
 60
 C0
-7C
-C6
-C6
+10
+38
+6C
 C6
 C6
 FE
@@ -8559,9 +8559,9 @@
 BITMAP
 00
 00
-7C
-C6
-C6
+10
+38
+6C
 C6
 C6
 FE
@@ -10365,9 +10365,9 @@
 BITMAP
 00
 00
-7C
-C6
-C6
+10
+38
+6C
 C6
 C6
 FE
@@ -12928,8 +12928,8 @@
 6C
 38
 00
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -12970,8 +12970,8 @@
 6C
 6C
 00
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -13011,8 +13011,8 @@
 BITMAP
 00
 00
-7E
-D8
+3E
+78
 D8
 D8
 FE
diff -u terminus-font-4.49.1.orig/ter-u16b.bdf terminus-font-4.49.1/ter-u16b.bdf
--- terminus-font-4.49.1.orig/ter-u16b.bdf	2023-09-11 22:10:29.517643166 +0300
+++ terminus-font-4.49.1/ter-u16b.bdf	2023-09-11 22:10:51.241013327 +0300
@@ -815,9 +815,9 @@
 BITMAP
 00
 00
-7C
-C6
-C6
+10
+38
+6C
 C6
 C6
 FE
@@ -2978,8 +2978,8 @@
 30
 18
 00
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -3001,8 +3001,8 @@
 18
 30
 00
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -3024,8 +3024,8 @@
 38
 6C
 00
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -3047,8 +3047,8 @@
 76
 DC
 00
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -3070,8 +3070,8 @@
 6C
 6C
 00
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -3093,8 +3093,8 @@
 38
 6C
 38
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -3115,8 +3115,8 @@
 BITMAP
 00
 00
-7E
-D8
+3E
+78
 D8
 D8
 FE
@@ -4449,9 +4449,9 @@
 BITMAP
 7C
 00
-7C
-C6
-C6
+10
+38
+6C
 C6
 C6
 FE
@@ -4496,8 +4496,8 @@
 6C
 38
 00
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -4541,9 +4541,9 @@
 BITMAP
 00
 00
-7C
-C6
-C6
+10
+38
+6C
 C6
 C6
 FE
@@ -7624,8 +7624,8 @@
 6C
 38
 00
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -7807,8 +7807,8 @@
 BITMAP
 7C
 00
-7E
-D8
+3E
+78
 D8
 D8
 FE
@@ -8199,8 +8199,8 @@
 0C
 18
 00
-7E
-D8
+3E
+78
 D8
 D8
 FE
@@ -9164,9 +9164,9 @@
 BITMAP
 60
 C0
-7C
-C6
-C6
+10
+38
+6C
 C6
 C6
 FE
@@ -9371,9 +9371,9 @@
 BITMAP
 00
 00
-7C
-C6
-C6
+10
+38
+6C
 C6
 C6
 FE
@@ -11349,9 +11349,9 @@
 BITMAP
 00
 00
-7C
-C6
-C6
+10
+38
+6C
 C6
 C6
 FE
@@ -14156,8 +14156,8 @@
 6C
 38
 00
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -14202,8 +14202,8 @@
 6C
 6C
 00
-7C
-C6
+38
+6C
 C6
 C6
 FE
@@ -14247,8 +14247,8 @@
 BITMAP
 00
 00
-7E
-D8
+3E
+78
 D8
 D8
 FE
diff -u terminus-font-4.49.1.orig/ter-u18b.bdf terminus-font-4.49.1/ter-u18b.bdf
--- terminus-font-4.49.1.orig/ter-u18b.bdf	2023-09-11 22:10:29.517643166 +0300
+++ terminus-font-4.49.1/ter-u18b.bdf	2023-09-11 22:10:51.244346666 +0300
@@ -884,9 +884,9 @@
 0000
 0000
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -3234,9 +3234,9 @@
 1800
 0C00
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -3259,9 +3259,9 @@
 0600
 0C00
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -3284,9 +3284,9 @@
 1E00
 3300
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -3309,9 +3309,9 @@
 3B80
 6E00
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -3334,9 +3334,9 @@
 3300
 3300
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -3359,9 +3359,9 @@
 1E00
 3300
 1E00
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -3384,9 +3384,9 @@
 0000
 0000
 0000
-7FC0
-C600
-C600
+1FC0
+3600
+6600
 C600
 C600
 FFC0
@@ -4834,9 +4834,9 @@
 0000
 3F00
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -4884,9 +4884,9 @@
 3300
 1E00
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -4934,9 +4934,9 @@
 0000
 0000
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -8284,9 +8284,9 @@
 3300
 1E00
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -8484,9 +8484,9 @@
 0000
 3F80
 0000
-7FC0
-C600
-C600
+1FC0
+3600
+6600
 C600
 C600
 FFC0
@@ -8909,9 +8909,9 @@
 0600
 0C00
 0000
-7FC0
-C600
-C600
+1FC0
+3600
+6600
 C600
 C600
 FFC0
@@ -9959,9 +9959,9 @@
 0000
 6000
 C000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -10184,9 +10184,9 @@
 0000
 0000
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -12334,9 +12334,9 @@
 0000
 0000
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -15384,9 +15384,9 @@
 3300
 1E00
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -15434,9 +15434,9 @@
 3300
 3300
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -15484,9 +15484,9 @@
 0000
 0000
 0000
-7FC0
-C600
-C600
+1FC0
+3600
+6600
 C600
 C600
 FFC0
diff -u terminus-font-4.49.1.orig/ter-u20b.bdf terminus-font-4.49.1/ter-u20b.bdf
--- terminus-font-4.49.1.orig/ter-u20b.bdf	2023-09-11 22:10:29.520976505 +0300
+++ terminus-font-4.49.1/ter-u20b.bdf	2023-09-11 22:27:56.532730540 +0300
@@ -952,9 +952,9 @@
 0000
 0000
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -3490,9 +3490,9 @@
 1800
 0C00
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -3517,9 +3517,9 @@
 0600
 0C00
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -3544,9 +3544,9 @@
 1E00
 3300
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -3571,9 +3571,9 @@
 3B80
 6E00
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -3598,9 +3598,9 @@
 3300
 3300
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -3625,9 +3625,9 @@
 1E00
 3300
 1E00
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -3652,9 +3652,9 @@
 0000
 0000
 0000
-7FC0
-C600
-C600
+1FC0
+3600
+6600
 C600
 C600
 C600
@@ -5218,9 +5218,9 @@
 0000
 3F00
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -5272,9 +5272,9 @@
 3300
 1E00
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -5326,9 +5326,9 @@
 0000
 0000
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -8944,9 +8944,9 @@
 3300
 1E00
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -9160,9 +9160,9 @@
 0000
 3F80
 0000
-7FC0
-C600
-C600
+1FC0
+3600
+6600
 C600
 C600
 C600
@@ -10753,9 +10753,9 @@
 0000
 6000
 C000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -10996,9 +10996,9 @@
 0000
 0000
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -13318,9 +13318,9 @@
 0000
 0000
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -16612,9 +16612,9 @@
 3300
 1E00
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -16666,9 +16666,9 @@
 3300
 3300
 0000
-3F00
-6180
-6180
+0C00
+1E00
+3300
 6180
 6180
 6180
@@ -16720,9 +16720,9 @@
 0000
 0000
 0000
-7FC0
-C600
-C600
+1FC0
+3600
+6600
 C600
 C600
 C600
Common subdirectories: terminus-font-4.49.1.orig/uni and terminus-font-4.49.1/uni
Common subdirectories: terminus-font-4.49.1.orig/win32 and terminus-font-4.49.1/win32
